tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -731,6 +731,95 @@ def test_log_null_metric(self):
             warnings.resetwarnings()
         assert exception_context.exception.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
 
+    def test__log_metrics(self):
```

Comment:
I didn't modify any test cases regarding log_batch, but I'll try to add more cases for it :)

Reply:
Ah, I think it would be great if we could write test cases for `log_batch()` and remove the test cases for the private methods. I.e. replace private method testing with public method testing, as long as we can get the same level of coverage.

Reply:
I revisited all unit test cases and converted them to call log_batch instead of private methods